iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -1265,6 +1277,19 @@ static int setMqttMessagePropertyIfPossible(IOTHUB_MESSAGE_HANDLE IoTHubMessage,
     // Not finding a system property to map to isn't an error.
```

Comment:
actually this function is confusing. should `propName` be a `const char**`?

Reply:
no, it's just a pointer to the property name string